client/html/pages/project/page.html

```diff
@@ -1,19 +1,16 @@
-<div class="page" ng-controller="mainCtrl">
```

Comment:
Don't add logic into /html folder - create menu template in angular app folders!